TARS/src/mcu_main/telemetry.h

```diff
@@ -68,6 +69,25 @@ struct TelemetryPacket {
     int16_t barometer_temp;   //[-128, 128]
 };
 
+struct CompactTelemetryPacket {
+    unsigned short idx = 1;
+    float gps_lat;
+    float gps_long;
+    float gps_alt;
+    uint8_t voltage_battery;  //[0, 16]
+    int16_t barometer_temp;   //[-128, 128]
+};
+
+struct Packets {
```

Comment:
Clarify the reasoning for having both possible messages defined in this singular struct in a comment. Another possibility would just be to separate the structs completely, but this does seem nice so a comment describing the design would help.